src/dnddiceroller/DiceTower.java

```diff
@@ -17,12 +19,33 @@
 public class DiceTower {
   final int PANEL_COUNT = 3;
   List<Die> dice;
+  int trayValue;
 
   public DiceTower() {
 	this.dice = new ArrayList();
+        int trayValue=0;
```

Comment:
you are re-initializing trayValue in the constructors scope. It does not assign the field.